_sass/modal.scss

```diff
@@ -39,5 +39,10 @@
         cursor: pointer;
         color: $lightgreen;
     }
+
+    /* video title */
+    .video{
+        color: var(--color-white);
```

Comment:
Can we give this className as `video-title` instead of just `video`? As it will be better specific className. Plus this will reduce the effort to write a comment above the className which you wrote. 
What do you think?

Reply:
Sure, I understand your suggestion to use `video-title` as the className instead of `video` for better specificity. I considered it, but since `video` was already assigned as the className, I decided to keep it consistent with the existing code.

Reply:
Seems fair enough, 
Let's wait for others opinion on this too.

Reply:
This is a good item to add to the weekly [Websites meeting agenda](https://docs.google.com/document/d/1XczAHXVe2FIWPqiF57ospJ43zw5cZQ7ui8mn39v5EvA/edit). You can add this item in the doc, attend, and present it. Meeting details at https://meshery.io/calendar.

Reply:
Thank you @leecalcote for sharing the meeting details. I have already included it in the agenda.